src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster.component.html

```diff
@@ -50,6 +51,15 @@ <h4 class="mt-3">Upgrade your current cluster to a multi-cluster setup effortles
 <div class="container-fluid h-100 p-4"
      *ngIf="isMultiCluster; else emptyCluster">
   <ng-container *ngIf="!loading; else loadingTpl">
+    <cd-alert-panel type="info"
+                    spacingClass="mb-3"
+                    [showTitle]="false"
+                    size="slim"
+                    *ngIf="showMessage"
+                    (dismissed)="showMessage = false"
+                    [dismissible]="true">
+      <p>Please note that the data for the disconnected cluster will remain accessible for a duration of around 5 minutes. After this period, it will be automatically removed.</p>
```

Comment:
let's be precise here with the duration.
```suggestion
      <p>Please note that the data for the disconnected cluster will be visible for a duration of 5 minutes. After this period, it will be automatically removed.</p>
```

Reply:
btw, this message will show up on every refresh if I am not wrong? or does it go away completely after you dismiss it. We want the latter. Can we configure it like that?

Reply:
I don't really get the need to show this message here, is not enought to have it in the modal for cluster discconection?

Also, don't forget to add internationalization (i18n)

Reply:
Thanks @nizamial09 , Updated the PR. @Pegonzal , we want to show this message here so that the user doesn't get confused seeing wrong data, so just to double check the behavior this was added.